backend/tracim_backend/error.py

```diff
@@ -62,6 +62,7 @@ class ErrorCode(IntEnum):
     EXTERNAL_AUTH_USER_PASSWORD_MODIFICATION_UNALLOWED = 2049
     USER_AUTH_TYPE_DISABLED = 2050
     WORKSPACE_AGENDA_DISABLED = 2051
+    FILE_TEMPLATE_NOT_AVAILABLE = 2052
```

Comment:
I feel like this error code is related to collaborative_document_edition code, so it should be in the same place

Reply:
Currently error are all stored in same enum, can change this but may be very complicated to avoid code number duplication, and for frontend to know what is the naming of error "2052".